app/src/main/res/values/strings.xml

```diff
@@ -281,8 +281,8 @@
   <string name="upload_problem_image">Problems found in image</string>
   <string name="internet_downloaded">Please only upload pictures that you have taken by yourself. Don\'t upload pictures that you have downloaded from the Internet.</string>
   <string name="give_permission">Give permission</string>
-  <string name="use_external_storage">Use external storage</string>
-  <string name="use_external_storage_summary">Save pictures taken with the in-app camera on your device</string>
+  <string name="use_external_storage">Save pictures</string>
+  <string name="use_external_storage_summary">Save pictures taken with the in-app camera to your device's disk</string>
```

Comment:
```suggestion
  <string name="use_external_storage_summary">Save pictures taken with the in-app camera to your device storage</string>
```

Reply:
All this is getting too twisty for me to click any commit buttons correctly. So you do that, please.
However maybe
- Save in-app shots,  should be
- Save In-app shots, capitalized.